music21/musicxml/m21ToXml.py

```diff
@@ -6552,6 +6617,29 @@ def testFromSoundingPitch(self):
         self.assertEqual(len(root.findall('.//transpose')), 1)
         self.assertEqual(root.find('.//step').text, 'D')
 
+    def testMultipleInstruments(self):
```

Comment:
what are the "Multiple Instruments" being tested?  I'm not really clear on this.

Reply:
Currently music21 only exports the first instrument in a stream.Part to musicXML. There are a handful of code comments referring to this, but nowhere in docs, I don't think.  I'll point out the existing implementation with a couple comments in this diff.

Reply:
i.e., you cannot export a mid-part Instrument change ("to Oboe" etc.)